src/__tests__/custom-functions/samples/streaming.txt

```diff
@@ -0,0 +1,118 @@
+// should pass streaming functions tests
+
+/**
+* increments a value each iteration
+* @param increment - number to add
+* @CustomFunction
+*/
+function incrementValue(increment: number, caller: IInvocationContext<number>): void {
+    var result = 0;
+    setInterval(function(){
+         result += increment;
+         caller.setResult(result);
+    }, 1000);
+}
+
+/**
+* increments a value each iteration
+* @param increment - string to add
+* @CustomFunction
+* @cancelable
+*/
+function incrementValue(increment: string, caller: IInvocationContext<string>) {
+    var result = 0;
+    setInterval(function(){
+         result += increment;
+         caller.setResult(result);
+    }, 1000);
+}
+
+/**
+* increments a value each iteration
+* @param increment - string to add
+* @CustomFunction
+*/
+function incrementValue(increment: number[][], caller: IInvocationContext<number[][]>) {
```

Comment:
What about adding a test for a FAILED streaming thing.  I.e., a caller that returns an object, or a Promise?